### ISSUE TITLE
Strange result - continue on last thread 

### ISSUE THREAD

**Issue by clarklight** [jinglescode/time-series-forecasting-tensorflowjs#9]
The last thread on the strange result was closed, i am not sure if that was resolved, but i am also facing the same issue, so want to continue the thread. 

I simply click on this link https://jinglescode.github.io/time-series-forecasting-tensorflowjs/

1.) I clicked fetch Data 
2.) Window size - click compute SMA and draw chart
3.) Train data - train dataset size 80%, Epochs 25, learning rate 0.01, hidden LSTM layers 4 - the loss finishes at 0.0008373598102480173
4.) Click validate and the result comes at a flat line

**Comment by jinglescode** [comment]
This is the challenge of machine learning, that a model, out of the box, doesn't work for all kinds of data. There is a lot of parameter tuning involve. There may need to do more feature engineering. Or there is a need to change the model architecture. 

Doing a quick search, you can see that you are not the only one who faced this problem:

- [LSTM always predict same values for all inputs in the batch](https://discuss.pytorch.org/t/lstm-always-predict-same-values-for-all-inputs-in-the-batch/53082)
- [Outputs of LSTM Model all Very Similar or the Same](https://discuss.pytorch.org/t/outputs-of-lstm-model-all-very-similar-or-the-same/84370)
- [Pytorch RNN always gives the same output for multivariate time series](https://www.reddit.com/r/learnmachinelearning/comments/9zugw8/pytorch_rnn_always_gives_the_same_output_for/)
- [PyTorch regression is producing the same numbers as prediction](https://stackoverflow.com/questions/59506405/pytorch-regression-is-producing-the-same-numbers-as-prediction)

I do not have an answer for you, unfortunately. Some suggestions would be:
- try tweaking the normalize the data portion (`.div(tf.scalar(10)` might be causing problems [line 19 and 20](https://github.com/jinglescode/time-series-forecasting-tensorflowjs/blob/master/model.js))
- include more data features, maybe can include volume as well
- include more data features, maybe trading indicators such as MACD, RSI, Bollinger bands
- build another model architecture

**Comment by jinglescode** [comment]
@clarklight, I've updated the repo to fix the normalisation steps, do give it a try.
